raiden/network/proxies/payment_channel.py

```diff
@@ -121,8 +127,8 @@ def close(
         self.token_network.close(
             channel_identifier=self.channel_identifier,
             partner=self.participant2,
-            nonce=nonce,
             balance_hash=balance_hash,
+            nonce=nonce,
```

Comment:
@err508 Why did you change the position of the nonce argument here and below?

Reply:
I reordered the arguments according to the contracts functions arguments.

Reply:
I see but this is not needed. The arguments need to be the same only for the contract proxy call, for which they are and to which you did not add the `channel_identifier`.

Reply:
To elaborate a bit. The reason I say it's not needed is that perhaps these functions are used in other places which you have not seen and things broke for no reason.

In principle, when making any changes to a codebase aim for the minimal working changes as every change, no matter how small, can have unforeseen consequences

Reply:
As stated in the commit message, these changes are optional. I'm aware it might require to change a lot, but I also found it annoying to compare function signatures with signatures that inconsistently ordered inside and across repos.

Reply:
I understand the sentiment. But this is how software evolves and keeping all functions have the same argument order is just something that's not going to happen. No gains and too much time and effort to maintain.

Reply:
I'd like to "leave the code cleaner than it was", as we have to touch all functions anyhow

Reply:
> I'd like to "leave the code cleaner than it was"

You did not. You changed a function but you did not change all the places that function is called from. So you left the code broken and I had to [add additional fixes](https://github.com/raiden-network/raiden/pull/2035/commits/596353b06e75aad242f11f426f14e384f8aa2635#diff-295b7685bc1e9c921272d10d5028abb8R349) on top.

And we don't have to touch all functions. There was no need to touch the order of the arguments of the functions of the proxies themselves.

Anyway it's done and probably fixed now. I am not discussing this further. In the future you will keep all code changes minimal.